cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -44,7 +44,7 @@ EXTRA_ARGS += --timescale $(COCOTB_HDL_TIMEUNIT)/$(COCOTB_HDL_TIMEPRECISION)
 
 SIM_BUILD_FLAGS += -std=c++11
 
-COMPILE_ARGS += --vpi --public-flat-rw --prefix Vtop -o $(TOPLEVEL) -LDFLAGS "-Wl,-rpath,$(LIB_DIR) -L$(LIB_DIR) -lcocotbvpi_verilator"
+COMPILE_ARGS += --vpi --public-flat-rw --prefix Vtop -o $(TOPLEVEL) -LDFLAGS "-Wl,-rpath,$(shell cocotb-config --lib-dir) -L$(shell cocotb-config --lib-dir) -lcocotbvpi_verilator"
```

Comment:
I'm wondering if instead of `cocotb-config --lib-dir` it would be better to use `dirname $(cocotb-config --lib-name-path vpi verilator)`

Reply:
It is rather to make custom flow bit easier and not our makefiles. No strong opinion.

Reply:
Looks like Windows supports forward slashes at the API level, but cmd.exe doesn't; so it depends if the path goes in as a value, or if cmd.exe parses it first. I guess we shouldn't worry about this for now.